src/api/telemetry/TelemetryAPI.js

```diff
@@ -253,6 +253,10 @@ export default class TelemetryAPI {
       options.domain = this.openmct.time.getTimeSystem().key;
     }
 
+    if (!Object.hasOwn(options, 'timeContext')) {
```

Comment:
We should remove [the equivalent code in TelemetryConnections](https://github.com/nasa/openmct/blob/37b2660f27006ed5a29f8d2815c6284ab22cc7e6/src/api/telemetry/TelemetryCollection.js#L65), no? [It's duplicative now](https://github.com/nasa/openmct/blob/37b2660f27006ed5a29f8d2815c6284ab22cc7e6/src/api/telemetry/TelemetryCollection.js#L130).